src/Synth/OscilGen.cpp

```diff
@@ -608,7 +608,9 @@ void OscilGen::recalculateDefaultWaveTable(WaveTable * wt, bool fillWithZeroes)
         {
             // no FFT/IFFT required, it's a simple sine
             // (the currently selected base function is still sine)
-            getbasefunction(data[i][j].data());
+            for(int k = 0; k < synth.oscilsize; ++k) {
```

Comment:
Ah yep, that resolves the previous comment about the default wavetable calculation, now this is pretty much independent of the rest of the oscil data.